modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationEndpoints/EditAPIApplicationEndpoint.testcase

```diff
@@ -35,6 +35,67 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test CanChangeSchemaOfEndpoint {
+		property portal.acceptance = "true";
+
+		task ("Given published API Application with Endpoint created") {
+			EndpointAPI.createAPIEndpoint(
+				apiApplicationERC = "My-app",
+				name = "testendpoint1",
+				path = "/testendpoint1");
+		}
+
+		task ("two API Schema in API Application") {
+			SchemaAPI.createAPISchema(
+				apiApplicationERC = "My-app",
+				mainObjectDefinitionERC = "L_API_APPLICATION",
+				name = "testSchema");
+		}
+
+		task ("When modifying endpoint again with API Schema schema2") {
+			APIBuilderUI.addAndPublishEndpointConfiguration(
```

Comment:
The steps now are update schema for different endpoints, and what we want to test is edit schema for same endpoint, please update the codes accordingly, thanks